src/server/scripts/EasternKingdoms/eastern_kingdoms_script_loader.cpp

```diff
@@ -15,7 +15,7 @@
  * with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-// This is where scripts' loading functions should be declared:
+ // This is where scripts' loading functions should be declared:
```

Comment:
Even though it may look better to align the comment marker with the asterisk column above, it is not related. Please note the start of the comment marker in line 1, column 1. We do not need to add a change here.